app/controllers/BarsController.scala

```diff
@@ -20,10 +20,11 @@ import config.FrontendAppConfig
 import connector.BackendConnector
 import javax.inject._
 import models._
-import play.api.i18n.I18nSupport
-import play.api.mvc._
 import play.api.Logger
-import uk.gov.hmrc.http.HeaderCarrier
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, _}
```

Comment:
Curious why the Action import is necessary when already importing _

Reply:
Removed the redundant Action import.